ultimarc/tools/mini_pac.py

```diff
@@ -71,7 +72,7 @@ def run():
     parser = ToolContextManager.get_argparser(tool_cmd, tool_desc)
 
     # TODO:  Setup additional program arguments here.
-
+    parser.add_argument('--read-config', help=_('Read Mini-pac device config'), default=False, action='store_true')
     parser.add_argument('--set-config', help=_('Set Mini-pac device config from config file'), type=str, default=None,
                         metavar='CONFIG-FILE')
     args = parser.parse_args()
```

Comment:
Seems like mixed terminology.  Should it be `--get-config` to go with `--set-config`.  Or you could change `--set-config` to `--write-config`.

Reply:
I lean towards you changing `--read-config` to `--get-config`.  `--write-config` seems like it would be confusing, would I be writing to the device or writing the device config to an output file?

Reply:
Changed read-config to get-config.  Fixing mixed terminology.